x-pack/plugins/security_solution/common/search_strategy/eql/validation/helpers.ts

```diff
@@ -4,8 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import get from 'lodash/get';
-import has from 'lodash/has';
+import { get, has } from 'lodash';
```

Comment:
For posterity, we're going back to named imports from defaults: 

https://github.com/elastic/kibana/pull/74539 -> https://github.com/elastic/kibana/pull/78156